src/main/resources/com/dataintuitive/viash/platforms/nextflow/VDSL3Helper.nf

```diff
@@ -91,7 +91,8 @@ def processDirectives(Map drctv) {
       def m = drctv["container"]
       assertMapKeys(m, [ "registry", "image", "tag" ], ["image"], "container")
       def part1 = 
-        params.containsKey("override_container_registry") ? params["override_container_registry"] + "/" : 
+        System.getenv('OVERRIDE_CONTAINER_REGISTRY') ? System.getenv('OVERRIDE_CONTAINER_REGISTRY') + "/" : 
```

Comment:
Should we prefix the env variable with `VIASH_`?

Reply:
Not making a statement on whether it should be prefixed or not, but that line should not have been there as it was already present on `develop`.